api.planx.uk/send/uniform.test.ts

```diff
@@ -33,20 +33,42 @@ jest.mock("@opensystemslab/planx-document-templates", () => {
   };
 });
 
-const mockGenerateOneAppXML = jest.fn().mockResolvedValue("<dummy:xml></dummy:xml>");
-const mockGenerateCSVData = jest.fn().mockResolvedValue([{ question: "Test", responses: [{ value: "Answer" }], metadata: {}}]);
 jest.mock("@opensystemslab/planx-core", () => {
   return {
     Passport: jest.fn().mockImplementation(() => ({
       getFiles: jest.fn().mockImplementation(() => []),
     })),
-    CoreDomainClient: jest.fn().mockImplementation(() => ({
+  };
+});
+
+const mockGenerateOneAppXML = jest
+  .fn()
+  .mockResolvedValue("<dummy:xml></dummy:xml>");
+
+jest.mock("../client", () => {
+  return {
+    _admin: {
       generateOneAppXML: () => mockGenerateOneAppXML(),
-      getDocumentTemplateNamesForSession: jest.fn().mockResolvedValue(["X", "Y"]),
-      getSessionById: () => jest.fn().mockResolvedValue(mockLowcalSession),
-      generateCSVData: () => mockGenerateCSVData(),
-    }))
-  }
+      getDocumentTemplateNamesForSession: jest
+        .fn()
+        .mockResolvedValue(["X", "Y"]),
+      getSessionById: jest.fn((id: string) => {
+        if (id === "noGeoJSON") {
+          const mockLowcalSessionCopy: any = { ...mockLowcalSession };
+          delete mockLowcalSessionCopy.data.passport.data[
+            "property.boundary.site"
+          ];
+          return mockLowcalSessionCopy;
+        }
+        return mockLowcalSession;
+      }),
+      generateCSVData: jest
+        .fn()
+        .mockResolvedValue([
+          { question: "Test", responses: [{ value: "Answer" }], metadata: {} },
+        ]),
+    },
+  };
```

Comment:
:bulb: super helpful to see/understand the `client` mocked like this, thank you for writing out !